python/ray/rllib/example.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
Hm I think it helps it stand out more as a file you're supposed to run, but don't have a strong preference either way.